docs/connecting-cloud-storage/amazon-s3.md

```diff
@@ -96,7 +96,10 @@ To configure CORS access, navigate to your S3 bucket inside your AWS console and
   {
     "AllowedHeaders": ["*"],
     "AllowedMethods": ["GET"],
-    "AllowedOrigins": ["https://app.kolena.io"],
+    "AllowedOrigins": [
+      "https://app.kolena.com",
+      "https://app.kolena.io"
+    ],
```

Comment:
Should we only document `https://app.kolena.com` so that new integrations do not involve `.io`

Reply:
I raised this after the sprint kickoff meetings too - we decided to keep both here for a short period of time during the migration (say 1-2 weeks) in case of issues.